ext/flash/01_http.js

```diff
@@ -569,16 +569,19 @@ function createServe(opFn) {
             );
 
             let resp;
+            let remoteAddr;
             try {
-              resp = handler(req, () => {
-                const { 0: hostname, 1: port } = core.ops.op_flash_addr(
-                  serverId,
-                  i,
-                );
-                return {
-                  hostname,
-                  port,
-                };
+              resp = handler(req, {
+                remoteAddr() {
```

Comment:
Missing `get`? Type suggests plain value but code is a function call.

Reply:
```suggestion
                get remoteAddr() {
```